### PR TITLE
Update Builder Documentation

### PR DESCRIPTION
**Pull request by kagarmoe**
To build the docs locally:

`cd www` 
`bundle exec middleman serve`

Signed-off-by: kagarmoe <kgarmoe@chef.io>

### CODE REVIEW

**Comment by kagarmoe** [comment]
- [ ] Create an origin
- [ ] origin settings
- [ ] origin secrets
- [ ] origin keys
- [ ] origin membership
- [ ] integrations
- [ ] upload and promote packages
- [ ] build jobs
- [ ] package secrets
- [ ] builder on prem
- [ ] revise data/docs nav
- [ ] fix any broken links
- [ ] copy to hugo component
- [ ] revise to new layout
- [ ] hugo nav
- [ ] list links for redirects hugo

File: www/source/docs/using-builder.html.md.erb [hunk 1]
```diff
@@ -236,59 +360,248 @@ Initialize the setting from the command line with:
...
+The syntax for the `set` subcommand is:
+
+```bash
+hab origin rbac set [FLAGS] [OPTIONS] <MEMBER_ACCOUNT> <ROLE> --origin <ORIGIN>
+```
+
+Set a member's RBAC role with:
+
+```bash
+hab origin rbac set bluewhale admin --origin two-tier-app
```
Comment (jeremymv2) [inline www/source/docs/using-builder.html.md.erb:603]: This should be changed to `administrator`

```
<ROLE>              The role name to enforce for the member account [possible values: readonly_member, member,
                        maintainer, administrator, owner]
```

File: www/source/docs/using-builder.html.md.erb [hunk 2]
```diff
@@ -236,59 +360,248 @@ Initialize the setting from the command line with:
...
+
+* [Download the Chef Habitat CLI](/docs/install-habitat)
+* [Create a Chef Habitat Builder account](/#builder-account)
+* [Generate a personal access token](/#builder-token)
+* [Create an origin](/#create-origin)
+* [Get origin keys](/#origin-keys)
+
+### Role-Based Access Control (RBAC) for Chef Habitat Builder (SaaS and on-prem)
+
+TODO:New in: SEMVER
```
Comment (jeremymv2) [inline www/source/docs/using-builder.html.md.erb:466]: Is this intentional?

Comment (kagarmoe) [reply www/source/docs/using-builder.html.md.erb:466]: Yes, I need to replace it (and write a Buildkite lint that looks for my todos, but only in docs files)

File: www/source/docs/using-builder.html.md.erb (hunk 1 above)
Comment (kagarmoe) [reply www/source/docs/using-builder.html.md.erb]: Fixed.

File: www/source/docs/using-builder.html.md.erb (hunk 2 above)
Comment (kagarmoe) [reply www/source/docs/using-builder.html.md.erb]: Fixed.

File: www/source/docs/using-builder.html.md.erb [hunk 3]
```diff
@@ -339,6 +675,10 @@ You can upload keys for the origin through the web interface for Builder, or by
 
 ### Upload Packages to Builder
 
+The purpose of Chef Habitat Builder is
```
Comment (jeremymv2) [inline www/source/docs/using-builder.html.md.erb:678]: Seems to me an unfinished thought perhaps?

Comment (kagarmoe) [reply www/source/docs/using-builder.html.md.erb:678]: An unfinished thought! Thanks for finding it!

**Pull request merged**